packages/block-editor/README.md

```diff
@@ -483,5 +483,4 @@ Undocumented declaration.
 
 <!-- END TOKEN(Autogenerated API docs) -->
 
-
```

Comment:
I've seen this around locally as well sometimes. Why does this happen?

Reply:
It's this because of https://github.com/WordPress/gutenberg/issues/15626 I'm investigating this.

Reply:
https://github.com/WordPress/gutenberg/pull/15679 should fix it.

Reply:
No I actually had to add empty lines to the previous npm release (because of the failure we had) to force learna to detect these as changed. This just reverts those unnecessary changes.

Reply:
> No I actually had to add empty lines to the previous npm release (because of the failure we had) to force learna to detect these as changed. This just reverts those unnecessary changes.

Can someone clarify for me: Do these changes matter? Is there anything further which needs to be done in this branch for them?

Reply:
No, nothing left to do here.